src/load/azext_load/data_plane/load_test/help.py

```diff
@@ -37,6 +37,9 @@
         az load test create --test-id sample-test-id --load-test-resource sample-alt-resource --resource-group sample-rg --display-name "Sample Name" --autostop-error-rate 80.5 --autostop-time-window 120
         az load test create --test-id sample-test-id --load-test-resource sample-alt-resource --resource-group sample-rg --display-name "Sample Name" --autostop disable
         az load test create --test-id sample-test-id --load-test-resource sample-alt-resource --resource-group sample-rg --display-name "Sample Name" --autostop enable
+    - name: Create a test with multi-region load configuration. The region name should be of format accepted by ARM, and should be a region supported by Azure Load Testing. Multi-region load tests can only be run against public endpoints.
```

Comment:
@saloniagrawal1997 please help review

Reply:
Create a test with a multi-region load configuration using region names in the format accepted by Azure Resource Manager (ARM). Ensure the specified regions are supported by Azure Load Testing. Multi-region load tests are restricted to public endpoints only.